rosbag2_storage_default_plugins/src/rosbag2_storage_default_plugins/sqlite/sqlite_wrapper.cpp

```diff
@@ -40,39 +44,14 @@ SqliteWrapper::~SqliteWrapper()
   sqlite3_close(db_ptr);
 }
 
-void SqliteWrapper::execute_query(const std::string & query)
+SqliteStatement SqliteWrapper::prepare_statement(std::string query)
```

Comment:
why the change from `const std::string &` ?

Reply:
No change, this is just git messing up. But changed to const ref.